client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -233,11 +237,16 @@ class ImageEditorCanvas extends Component {
 			canvasX = -50 * widthRatio - 100 * leftRatio,
 			canvasY = -50 * heightRatio - 100 * topRatio;
 
+		const canvasOffsetTop = canvas.offsetTop;
+		const canvasOffsetLeft = canvas.offsetLeft;
+		const canvasOffsetWidth = canvas.offsetWidth;
+		const canvasOffsetHeight = canvas.offsetHeight;
+
```

Comment:
Yes @gwwar, I know that. What I say if we will create these constants only to make the code more clear maybe we could simplify it even more.